modules/ipi-install-troubleshooting-post-install-pod-errors.adoc

```diff
@@ -0,0 +1,175 @@
+[id="ipi-install-troubleshooting-post-install-pod-errors_{context}"]
+
+= Post install Pod errors
+
+Use the following procedure to troubleshoot `metal3` Pod errors during deployment.
+
+.Procedure
+
+. Retrieve the status for the `openshift-machine-api`:
++
+[source,bash]
+----
+[kni@provisioner ~]$ oc get all -n openshift-machine-api
+----
++
+----
+NAME                         READY   STATUS
+pod/metal3-6c6cc7c56c-lj4lr  0/8     Init:CreateContainerConfigError
+----
++
+A `CreateContainerConfigError` occurs if there is no `ConfigMap`.
++
+----
+  Warning  Failed     <invalid> (x7 over 32s)  kubelet, master-1.<cluster-name>.example.com  Error: configmap "metal3-config" not found
+----
++
+Ensure the `metal3-config.yaml` file has a `ConfigMap` section. If there is no `metal3-config.yaml`, create one in the subsequent steps.
+
+. Create the `metal3-config.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: metal3-config
+  namespace: openshift-machine-api
+data:
+  cache_url: rhcos-43.81.202001142154.0-qemu.x86_64.qcow2.gz
+  deploy_kernel_url: http://172.22.0.1:6180/images/ironic-python-agent.kernel
+  deploy_ramdisk_url: http://172.22.0.1:6180/images/ironic-python-agent.initramfs
+  dhcp_range: 172.22.0.10,172.22.0.100
+  http_port: "6180"
+  ironic_endpoint: http://172.22.0.1:6385/v1/
+  ironic_inspector_endpoint: http://172.22.0.3:5050/v1/
+  provisioning_interface: eno1
+  provisioning_ip: 172.22.0.1/24
+  rhcos_image_url: <URL-which-has-qcow-image>
+----
++
+*Note*: Change the `rhcos_image_url` to the appropriate URL for the
```

Comment:
[NOTE]
    ====
    Change the `rhcos_image_url` to the appropriate URL for the deployment environment.
    ====